app/models/mail_server_log.rb

```diff
@@ -23,7 +23,7 @@ class MailServerLog < ActiveRecord::Base
 
   serialize :delivery_status, DeliveryStatusSerializer
 
-  before_create { |record| record.delivery_status }
```

Comment:
might have been helped along by the confusion between `display_status` being a method and an db attribute